xivo_dao/resources/switchboard/tests/tests_dao.py

```diff
@@ -61,6 +61,20 @@ def test_search_multi_tenant(self):
         tenants = [tenant.uuid]
         self.assert_search_returns_result(expected, tenant_uuids=tenants)
 
+    def test_search_by_extension(self):
+        switchboard = self.add_switchboard()
+        incall = self.add_incall()
+        self.add_extension(type='incall',
+                           typeval=str(incall.id),
+                           exten="1000")
+        self.add_dialaction(category='incall',
+                            categoryval=str(incall.id),
+                            action='switchboard',
+                            actionarg1=switchboard.uuid)
+
+        expected = SearchResult(1, [switchboard])
+        self.assert_search_returns_result(expected, search="1000")
```

Comment:
add test `exten=..` arg
```suggestion
        self.assert_search_returns_result(expected, search="1000")
        self.assert_search_returns_result(expected, exten="1000")
```